### PR TITLE
Consistent layout (#247)

### LINKED ISSUE #247: As a user I want a consistent layout
**Description:**

A user want to see a consistent layout for a clean UI.

**Acceptance criteria:**
- [x] All HTML tags use [Bootstrap](https://bootstrap-vue.js.org/docs/components) if possible
- [x] Input validation
- [x] No inline CSS

**Test case (Web browser):**
- [x] Check if all HTML tags use valid Bootstrap

### PR DESCRIPTION
**Pull request by Simon-Deuring**
### New
- Consistent layout for all views
### Changes
- All HTML tags use Bootstrap if possible
- Input validation for all fields
- No inline CSS
- Describing comments for all views, components and methods

### CODE REVIEW

File: vue/src/components/BoardSidebar.vue [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<!-- This sidebar displays all notice board accessable for the user -->
```
Comment (DevPhilB) [inline vue/src/components/BoardSidebar.vue:1]: accessable != accessible

File: vue/src/components/BoardSidebar.vue [hunk 2]
```diff
@@ -93,6 +94,7 @@ export default {
     this.loading = false
   },
   methods: {
+    // Used to load the all boards accessable for the user
```
Comment (DevPhilB) [inline vue/src/components/BoardSidebar.vue:97]: Cf. first comment (change it to accessible)

File: vue/src/components/NoteFooter.vue [hunk 3]
```diff
@@ -1,3 +1,4 @@
+<!-- This footer is displayed beneath every note. The deletion button only is shown if the user is authorized to delete it -->
```
Comment (DevPhilB) [inline vue/src/components/NoteFooter.vue:1]: "This footer is displayed under the content of every note. The deletion button is only shown if the user is authorized to delete it" or just "This footer is part of every note. The deletion button is only shown if the user is authorized to delete it"

File: vue/src/components/cms/PermissionPanel.vue [hunk 4]
```diff
@@ -1,51 +1,69 @@
+<!-- This panel shows a list of user rights. They can be granted and deprived separately -->
```
Comment (DevPhilB) [inline vue/src/components/cms/PermissionPanel.vue:1]: "This panel shows a list of user rights. They can be given and revoked separately"

File: vue/src/components/cms/UserList.vue [hunk 5]
```diff
@@ -1,3 +1,5 @@
+<!-- This panel shows a list of all registered users. For each user the profil page can be visited and edited.
+     It also allows to promote, demote and delte users -->
```
Comment (DevPhilB) [inline vue/src/components/cms/UserList.vue:2]: "This panel shows a list of all registered users. Each user profil can be visited directly in view- or edit-mode.
"It also allows to promote, demote and delete users"

File: vue/src/components/cms/UserList.vue [hunk 6]
```diff
@@ -322,33 +328,38 @@ export default {
...
+          editable: false
         }
       })
     },
-    clickDelete () {
-      this.viewContextMenu = false
-      this.deleteUser(this.selectedId)
-    },
-    clickProfile () {
+    // This method opens the profile page in editing mode
```
Comment (DevPhilB) [inline vue/src/components/cms/UserList.vue:346]: This method opens the profile page in edit mode

File: vue/src/components/editors/EditorIcon.vue [hunk 7]
```diff
@@ -1,3 +1,4 @@
+<!-- This class is used to draw the icons visible beneath the text note editor -->
```
Comment (DevPhilB) [inline vue/src/components/editors/EditorIcon.vue:1]: This class is used to draw the icons visible under the text note editor

**Review by DevPhilB — CHANGES_REQUESTED** [review]
Had to change the base branch to development 👎 
But in general, good job! 👍 
I've added some feedback (w/ solution) and as optional task you could change CSS class names, so that they are consistent too (it looks like you have already started it 😃 )

**Comment by Simon-Deuring** [comment]
I implemented all your suggestions and changed CSS class names to be standardized.
But I still think that "to grant rights" is better than to "to give rights".

**Review by DevPhilB — APPROVED** [review]
👍

**Pull request merged**